server/config/config_test.go

```diff
@@ -209,7 +209,7 @@ enable-remove-extra-replica = false
 	c.Assert(cfg.Schedule.EnableRemoveDownReplica, IsFalse)
 	c.Assert(cfg.Schedule.EnableMakeUpReplica, IsFalse)
 	c.Assert(cfg.Schedule.EnableRemoveExtraReplica, IsFalse)
-	b, err := json.Marshal(cfg)
+	b, err := json.Marshal(cfg.Schedule)
```

Comment:
Why need to update it?

Reply:
Please look the issue reported by the commands: 
```
GO111MODULE=on staticcheck ./...
```

Reply:
I think it's false positive report.

Reply:
> I think it's false positive report.

The idea is from this issue reported [https://github.com/pingcap/tidb/issues/13456](url)

Reply:
> I think it's false positive report.

siddontang said that CI will use the tool to validate the Go code.

Reply:
> I think it's false positive report.

No break is needed in the default case.

Reply:
I don't care what he said. There is nothing wrong with the code. The tool has bug to report it.

Reply:
> I don't care what he said. There is nothing wrong with the code. The tool has bug to report it.

OK! Please look the the comments of the func `json.Marshal`. It described as the following:
```
// Channel, complex, and function values cannot be encoded in JSON.
// Attempting to encode such a value causes Marshal to return
// an UnsupportedTypeError.
```

Now, `cfg` is a function.

Reply:
why is `cfg` a function?

Reply:
> why is `cfg` a function?

Maybe the tool has a bug , but what I changed doesn't affect the execution. Ignore the case if you think it is not problem.

Reply:
I think it's not a problem. Please revert this line.

Reply:
> I think it's not a problem. Please revert this line.

Done.